CORE_NAMING_GUIDELINE.md

```diff
@@ -96,16 +96,17 @@ concept IsGraph = requires(T t) {
 
 /** @brief Represents a simple graph structure for signals. (alt documentation)*/
 template <typename TBlock>
-class Graph : public Block<Graph<TBlock>, Doc<R"(
+class Graph : public Block<Graph<TBlock>> {
+    bool               _initialised = true; // private member using `_` prefix
```

Comment:
NB: we should update this example to not incentivize users to implement their own state machine on top of the one provided by the block.